3529/2/Lib/checkout_context.rb

```diff
@@ -0,0 +1,19 @@
+require 'fileutils'
+require './Lib/Utiles/path_loader.rb'
```

Comment:
Don't think you need this includes
But if so - include all files in this folder with 1 line